.github/workflows/tests-ci.yml

```diff
@@ -107,31 +115,35 @@ jobs:
       matrix:
         include:
           - group: "A"
-            packages: "--workspace packages/agent"
+            packages: "--filter agent"
           - group: "B"
-            packages: "--workspace packages/credentials --workspace packages/crypto --workspace packages/dids --workspace packages/proxy-agent --workspace packages/identity-agent --workspace packages/user-agent"
+            packages: "--filter credentials --filter crypto --filter dids --filter proxy-agent --filter identity-agent --filter user-agent"
           - group: "C"
-            packages: "--workspace packages/api --workspace packages/common"
+            packages: "--filter api --filter common"
     steps:
       - name: Checkout source
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
         with:
           submodules: true
+      
+      - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d
+        with:
+          version: 8
 
       - name: Set up Node.js
         uses: actions/setup-node@5ef044f9d09786428e6e895be6be17937becee3a #v4.0.0
         with:
           node-version: 18
           registry-url: https://registry.npmjs.org/
-          cache: "npm"
+          cache: "pnpm"
 
       - name: Install dependencies
-        run: npm ci
+        run: pnpm install --frozen-lockfile
 
       - name: Get Playwright Version (for cache)
         id: get-playwright-version
         run: |
-          PLAYWRIGHT_VERSION=$(npm ls @playwright/test --workspace=./packages/api | grep '@playwright/test' | awk 'NR==1{print $2}')
+          PLAYWRIGHT_VERSION=$(pnpm --filter agent ls @playwright/test | grep '@playwright/test' | awk 'NR==1{print $2}')
```

Comment:
@shamilovtim Not that it matters but should we use the same filter for both the `Get Playwright Version` and `Install Playwright Browsers` tasks?  It initially took me a bit of stare and compare to figure out whether it mattered if one was referencing `--filter agent` and the other `--filter api`.

Reply:
Yes I was being lazy to modify them to the same 😄

Reply:
Addressed in [4442df4](https://github.com/TBD54566975/web5-js/pull/419/commits/4442df4f47b676650cbb74fa85b0c48917fc600f)